### ISSUE TITLE
Output redirection doesn't work in aliases

### ISSUE THREAD

**Issue by Cheaterman** [xonsh/xonsh#1262]
Hello buddies, I hope you're all doing good ^__^

Apparently output redirection doesn't work in aliases:

```
cheaterman@cmans-laptop ~ $ aliases['bli'] = 'echo bli &> /dev/null'
cheaterman@cmans-laptop ~ $ bli
bli &> /dev/null
```

Thanks in advance for investigating/fixing ^__^

**Comment by BlaXpirit** [comment]
It's not specific to output redirection, it's just defined to space-separate and run the command with the exact arguments as written

**Comment by scopatz** [comment]
Thanks for reporting @Cheaterman!

**Comment by Cheaterman** [comment]
Thanks to you for acknowledging @scopatz ^__^

**Comment by scopatz** [comment]
Potential fix for this in #2250

**Comment by gforsyth** [comment]
Closed via #2250
